<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SQLite 1.1</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/images/SQLite_logo.ico</normaloff>:/images/SQLite_logo.ico</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>36</width>
    <height>36</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>900</width>
      <height>1000</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>50</y>
      <width>54</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>学号：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_studentID">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>50</y>
      <width>191</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_name">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>120</y>
      <width>191</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>120</y>
      <width>54</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>姓名：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>190</y>
      <width>54</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>性别：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>260</y>
      <width>54</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>年龄：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_age">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>260</y>
      <width>191</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>330</y>
      <width>54</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>成绩：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_score">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>330</y>
      <width>191</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_gender">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>190</y>
      <width>191</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>男</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>女</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_create">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>420</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>创建表</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_insert">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>520</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>插入记录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_delete">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>620</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>删除记录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_modify">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>720</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>修改记录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_select">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>820</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>查找记录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_findall">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>920</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>查找所有记录</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
